inst/WORDLIST

```diff
@@ -0,0 +1,4 @@
+HMC
```

Comment:
Why do you have HMC here while we don't deal with HMC yet?

Reply:
I mention HMC in the docs somewhere and it's flagged as a misspelling; this list is just words that we tell the spellchecker are ok